stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>This looks like it should be really neat, but I didn't get a widget when running in Jupyter Lab unfortunately. I'm not sure why I'm afraid (there were no errors, just some text displayed).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>We'll have to check if this works on the website version at all, there's a good chance we'll have to remove this.</p>